api/src/main/java/org/openmrs/module/bedmanagement/service/BedTagMapService.java

```diff
@@ -1,25 +1,55 @@
-package org.openmrs.module.bedmanagement;
+package org.openmrs.module.bedmanagement.service;
 
 import org.openmrs.annotation.Authorized;
 import org.openmrs.api.OpenmrsService;
+import org.openmrs.module.bedmanagement.entity.Bed;
+import org.openmrs.module.bedmanagement.entity.BedTag;
+import org.openmrs.module.bedmanagement.entity.BedTagMap;
 import org.openmrs.module.webservices.rest.web.response.IllegalPropertyException;
 import org.springframework.transaction.annotation.Transactional;
 
 @Transactional
 public interface BedTagMapService extends OpenmrsService{
 
+    /**
+     * Save / Update bedTagMap
+     * @param bedTagMap {@link BedTagMap}
+     * @return {@link BedTagMap}
+     * @throws IllegalPropertyException
+     */
     @Authorized(value = {"Edit Tags", "Get Tags", "Get Beds"}, requireAll=true)
     BedTagMap save(BedTagMap bedTagMap) throws IllegalPropertyException;
```

Comment:
Should be `saveTagMap(..)`. I know this was already there, but that's the occasion to refactor.